main/gps.go

```diff
@@ -260,20 +268,25 @@ func initGPSSerial() bool {
 		p.Write(makeUBXCFG(0x06, 0x24, 36, nav))
 
 		// GNSS configuration CFG-GNSS for ublox 7 higher, p. 125 (v8)
-		// NOTE: Max position rate = 5 Hz if GPS+GLONASS used.
-
-		// TESTING: 5Hz unified GPS + GLONASS
 
-		// Disable GLONASS to enable 10 Hz solution rate. GLONASS is not used
-		// for SBAS (WAAS), so little real-world impact.
+		// Notes: ublox8 is multi-GNSS capable (simultaneous decoding of GPS and GLONASS, or
+		// GPS and Galileo) if SBAS (e.g. WAAS) is unavailable. This may provide robustness
+		// against jamming / interference on one set of frequencies. However, this will drop the
+		// position reporting rate to 5 Hz during times multi-GNSS is in use. This shouldn't affect
+		// gpsattitude too much --  without WAAS corrections, the algorithm could get jumpy at higher
+		// sampling rates.
 
 		cfgGnss := []byte{0x00, 0x20, 0x20, 0x05}
 		gps := []byte{0x00, 0x08, 0x10, 0x00, 0x01, 0x00, 0x01, 0x01}  // enable GPS with 8-16 tracking channels
 		sbas := []byte{0x01, 0x02, 0x03, 0x00, 0x01, 0x00, 0x01, 0x01} // enable SBAS (WAAS) with 2-3 tracking channels
 		beidou := []byte{0x03, 0x00, 0x10, 0x00, 0x00, 0x00, 0x01, 0x01}
 		qzss := []byte{0x05, 0x00, 0x03, 0x00, 0x00, 0x00, 0x01, 0x01}
-		//glonass := []byte{0x06, 0x04, 0x0E, 0x00, 0x00, 0x00, 0x01, 0x01} // this disables GLONASS
-		glonass := []byte{0x06, 0x08, 0x0E, 0x00, 0x01, 0x00, 0x01, 0x01} // this enables GLONASS with 8-14 tracking channels
+		glonass := []byte{0x06, 0x04, 0x0E, 0x00, 0x00, 0x00, 0x01, 0x01} // this disables GLONASS
+
+		if (globalStatus.GPS_detected_type == GPS_TYPE_UBX8) || (globalStatus.GPS_detected_type == GPS_TYPE_UART) { // assume that any GPS connected to serial GPIO is ublox8 (RY835/6AI)
```

Comment:
ublox6 should ignore it. ublox7 should preferentially configure for GPS-only.

My 725AI is wired for USB at the moment so I can't test over UART, but I recompiled with `if (globalStatus.GPS_detected_type == GPS_TYPE_UBX7)...` to send the same commands over USB. This worked as expected, configuring for GPS + WAAS.